### PR TITLE
Develop->Master 0.1.5

### PR DESCRIPTION
**Pull request by Xiangs18**

### CODE REVIEW

**Comment by Xiangs18** [comment]
This is strange; it ran twice.

**Comment by MrCreosote** [comment]
What ran twice?

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+
+* The MongoDB clients have been updated to the most recent version.
+* Added the ``mongo-retrywrites`` configuration setting in ``deploy.cfg``, defaulting to
+  ``false``.
+
+## 0.1.4
+
+* Switched the build system from Ant to Gradle. As such, all build artifacts, including the
+  `assembly_homology` script, are now found in the `build` directory.
+
```
Comment (MrCreosote) [inline RELEASE_NOTES.md:13]: If 0.1.4 has never been released, we should just remove the 0.1.5 release and merge the notes

**Comment by Xiangs18** [comment]
Docker / Build
assembly_homology_tests (8, mongodb-linux-x86_64-ubuntu2204-7.0.4)
assembly_homology_tests (11, mongodb-linux-x86_64-ubuntu2004-4.4.29)

**Review by MrCreosote — COMMENTED** [review]
This has all been reviewed before, so just skimmed it

File: RELEASE_NOTES.md (hunk 1 above)
Comment (Xiangs18) [reply RELEASE_NOTES.md:13]: That's what I thought. I should have just added it to the 0.1.4 notes.

Comment (Xiangs18) [reply RELEASE_NOTES.md]: 👍

**Review by MrCreosote — APPROVED** [review]

**Pull request merged**